examples/gemm_sm100/gemm_tcgen5mma_ws_2sm.py

```
@@ -0,0 +1,211 @@
+
+import torch
+import tilelang
+import tilelang.language as T
+from tilelang.profiler import do_bench
+from tilelang.engine import register_cuda_postproc
+
+tilelang.disable_cache()
+
+
+@register_cuda_postproc
+def _(code, _):
```

Ruff (0.15.4)

[warning] Unused function argument: `code`

(ARG001)